CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
-## [2.0.5] - 12-02-22
+## [2.0.5-rc.2] - 09-13-24
+### Resolves
+
+- Downgrading Angular dependencies to 12.2.16 to fix npm build.
+- Added `"cheerio": "1.0.0-rc.10"` to fix npm build.
```

Comment:
This technically fixes the compodocs command from running correctly.

Reply:
Modified this line to incorporate your suggestion.